server/store.go

```diff
@@ -327,6 +327,28 @@ func (l *listStore) getList(userID, listID string) ([]*IssueRef, []byte, error)
 	return list, originalJSONList, nil
 }
 
+func (l *listStore) CountIssues(userID string) (*CountIssue, error) {
+	myList, err := l.GetList(userID, MyListKey)
+	if err != nil {
+		return nil, err
+	}
+	inList, err := l.GetList(userID, InListKey)
+	if err != nil {
+		return nil, err
+	}
+	outList, err := l.GetList(userID, OutListKey)
+	if err != nil {
+		return nil, err
+	}
+
+	return &CountIssue{
+		In:  len(inList),
+		My:  len(myList),
+		Out: len(outList),
+	}, nil
```

Comment:
Might as well just return the actual lists right? Then the frontend doesn't need to do any other requests